sdk/azcore/arm/runtime/policy_register_rp.go

```diff
@@ -176,6 +176,7 @@ var unregisteredRPCodes = []string{
 	"MissingSubscriptionRegistration",
 	"MissingRegistrationForResourceProvider",
 	"Subscription Not Registered",
+	"SubscriptionNotRegistered",
```

Comment:
Why are there both of these?

Reply:
Per the ARM spec, the value is supposed to _always_ be `MissingSubscriptionRegistration`.  Unfortunately, some RPs got it wrong (and can't fix it due to back-compat) which is why we now have four values.